src/core/control/settings/Settings.cpp

```diff
@@ -163,8 +166,10 @@ void Settings::loadDefault() {
     this->buttonConfig[BUTTON_STYLUS_TWO] =
             new ButtonConfig(TOOL_NONE, Colors::black, TOOL_SIZE_NONE, DRAWING_TYPE_DEFAULT, ERASER_TYPE_NONE);
 
-    this->fullscreenHideElements = "mainMenubar";
-    this->presentationHideElements = "mainMenubar,sidebarContents";
+    // view modes
+    this->activeViewMode = ViewMode::VIEW_MODE_DEFAULT;
+    this->viewModes = std::vector<std::string>{"default","fullscreen","presentation"};
+    this->viewModeAttributes =std::vector<std::string>{"showMenubar,showToolbar,showSidebar","showToolbar,showSidebar,goFullscreen","showToolbar,goFullscreen"};
```

Comment:
Formatting: Space after `=` and after `,` in argument list

Reply:
Probably happened when copy-pasting. Will fix it, so the formatting fits again.